Source/Krypton Components/Krypton.Toolkit/Designers/Other/KryptonSplitContainerBehavior.cs

```diff
@@ -75,16 +78,19 @@ public override bool OnMouseDown(Glyph? g, MouseButtons button, Point pt)
         /// <returns>true if the message was handled; otherwise, false.</returns>
         public override bool OnMouseMove(Glyph? g, MouseButtons button, Point pt)
         {
-            if (_splitContainer != null)
+            // Glyph g cannot be null, also not in base.OnMouseDown
+            if (g is not null && _splitContainer is not null)
             {
                 // Convert the adorner coordinate to the split container client coordinate
                 Point splitPt = PointToSplitContainer(g, pt);
 
                 // Notify the split container so it can track mouse message
                 _splitContainer.DesignMouseMove(splitPt);
+
+                return base.OnMouseMove(g, button, pt);
```

Comment:
Why is this inside the `if` ?
Should it not be on the outside and let the base sort it out ?

Reply:
base.OnMouseMove expects valid references/params and does not handle nulls
base is in class/assembly System.Windows.Forms.Design.Behavior.Behavior and of course cannot be altered.

Reply:
Place a comment and leave inside the if then.

Reply:
Comment added.